backend/mora/validator.py

```diff
@@ -150,27 +150,47 @@ def is_distinct_responsibility(
         )
 
 
-def is_date_range_in_employee_range(employee_uuid, valid_from, valid_to):
+def is_date_range_in_employee_range(employee_obj: dict,
+                                    valid_from, valid_to):
     scope = lora.Connector(
         virkningfra=util.to_lora_time(valid_from),
         virkningtil=util.to_lora_time(valid_to)
     ).bruger
-    employee = scope.get(employee_uuid)
+    # If this is a not-yet created user, emulate check
+    if employee_obj.get('future'):
+        employee_valid_from = employee_obj.get(mapping.VALID_FROM)
+        employee_valid_to = employee_obj.get(mapping.VALID_TO)
+        is_contained_in_employee_range(employee_valid_from, employee_valid_to,
+                                       valid_from, valid_to)
+    else:
+        employee_uuid = employee_obj.get(mapping.UUID)
+        employee = scope.get(employee_uuid)
+
+        if not employee:
+            raise exceptions.HTTPException(
+                exceptions.ErrorCodes.E_USER_NOT_FOUND,
+                employee_uuid=employee_uuid,
+            )
 
-    if not employee:
-        raise exceptions.HTTPException(
-            exceptions.ErrorCodes.E_USER_NOT_FOUND,
-            employee_uuid=employee_uuid,
-        )
+        gyldighed_key = "brugergyldighed"
 
-    gyldighed_key = "brugergyldighed"
+        if not _is_date_range_valid(employee, valid_from, valid_to, scope,
+                                    gyldighed_key):
+            raise exceptions.HTTPException(
+                exceptions.ErrorCodes.V_DATE_OUTSIDE_EMPL_RANGE,
+                employee_uuid=employee_uuid,
+                **_get_active_validity(employee),
+            )
 
-    if not _is_date_range_valid(employee, valid_from, valid_to, scope,
-                                gyldighed_key):
+
+def is_contained_in_employee_range(empl_from, empl_to, valid_from, valid_to):
+    if valid_from < empl_from or empl_to < valid_to:
```

Comment:
This check is probably correct, but I've gotten checks like this subtly wrong, so we should make sure to unit test it thoroughly.